rca/enquire_to_study/models.py

```diff
@@ -154,3 +156,37 @@ class EnquiryFormSubmissionProgrammesOrderable(Orderable):
     panels = [
         SnippetChooserPanel("programme"),
     ]
+
+
+@register_setting
```

Comment:
AC: As an administrator I can define copy and subject line for the email notification